item.rb

```diff
@@ -0,0 +1,25 @@
+require 'date'
+
+class Item
+  attr_accessor :publish_date, :label, :gener, :author
```

Comment:
- [OPTIONAL] It might be better to change :gener to :genre (a typo) to be sure that it would be consistent across the codebase and for later interaction with this codebase.

Reply:
Alright!!